pkg/selectors/selectors.go

```diff
@@ -61,6 +73,13 @@ type KernelSelectorState struct {
 	newBinVals    map[uint32]string              // these should be added in the names_map
 
 	listReader ValueReader
+	// stringMaps are used to populate string and char buf matches
+	stringMaps1 []map[[StringMapsSize1]byte]struct{}
+	stringMaps2 []map[[StringMapsSize2]byte]struct{}
+	stringMaps3 []map[[StringMapsSize3]byte]struct{}
+	stringMaps4 []map[[StringMapsSize4]byte]struct{}
+	stringMaps5 []map[[StringMapsSize5]byte]struct{}
+	stringMaps6 []map[[StringMapsSize6]byte]struct{}
```

Comment:
Do we need to reflect the exact structure of the bpf code? Can we just have string maps here and do the conversion when we load the bpf map? This means that we could also use a single map for everything, which I think would simplify the code.

Reply:
I thought a one-to-one mapping was easier to understand, and follows the style for (at least the majority, if not all) of the other maps.

Reply:
I've changed this to an array of maps as a good middleground.